test/runnable/extra-files/cdcmp.d

```diff
@@ -0,0 +1,143 @@
+
+bool test_ltz(ubyte x) { return x <  0; }
```

Comment:
Why no static foreach?

Reply:
Because I don't want the code to be clever - it's there to test the code generator. If I use foreach, then:

1. if I want to pull out the failing one into its own file, I have to reconstruct it from the foreach
2. by naming them this way, instead of generating names, it makes it easy to pick the test case out of the objdump file.

Note that I often have to do (1) and (2) when trying to isolate a problem.